### PR TITLE
Landing Page

### LINKED ISSUE #7: OSCAL-Reference needs a nice landing page.
We already talked about this, but did not see an issue anywhere.  Apologies if duplicated.  The initial page of the OSCAL reference could use a landing page-style format that gives an overview of the content and audience of the site.  We might assume some people will see this as their first encounter with OSCAL and provide a little guidance to the other locations for content.

Also noticed that we need the link set in Settings for the pages URL that displays on the github project.

### PR DESCRIPTION
**Pull request by aj-stein-nist**
Add landing page for usnistgov/OSCAL-Reference#7 with a revised approached, where /OSCAL-Reference/ redirects to an updated model versions page and explanatory text from the previous iteration of the landing page.

Screen Shot 2023-08-11 at 15 29 12-fullpage

### CODE REVIEW

**Review by wendellpiez — APPROVED** [review]
Nice work --

**Comment by aj-stein-nist** [comment]
Thanks for the approval. I talked to @iMichaela and I may consider an alternate approach, even if this works quite well, and make a redirect, adding the landing page material as-is in the current PR to the model reference page.

File: site/layouts/_default/reference-index.html [hunk 1]
```diff
@@ -50,6 +50,22 @@
...
+          <img src="/img/outline-ssp-json.png"/>
+          <img src="/img/outline-ssp-xml.png"/>
+          <p>In an outline, the cardinality, the minimum and maximum number of instances required, follows the entry as a number in square brackets. The data type follows the cardinality (see the cardinality <code>[1]</code> in the image below).</p>
+          <h2 id="reference-pages">Reference Pages</h2>
+          <p>In an outline, each entry links to its corresponding reference providing a definition for an entry and its constraints.</p>
+          <img src="/img/reference-ssp-json.png"/>
+          <h2 id="index-pages">Index Pages</h2>
+          <p>An index for each OSCAL model lists all entries in alphabetical order.</p>
+          <p>Each entry in the index lists a specific location in the applicable model, in both JSON and XML formats.</p>
+          <img src="/img/index-ssp-props-json.png"/>
```
Comment (aj-stein-nist) [inline site/layouts/_default/reference-index.html]: Concern: I am not sure this approach is maybe hacky and I am not sure how to make a special block to put in `content/models/_index.md` correctly while keeping the release number listing "in the middle" and not make expert users strain for novices or vice versa. Would love feedback.

Comment (aj-stein-nist) [reply site/layouts/_default/reference-index.html]: Nikita actually mentioned an interesting workaround for this that I will try with a file for additional content that is a trailer to the list, it is less kludgey than this.

Comment (aj-stein-nist) [reply site/layouts/_default/reference-index.html]: OK I finally figured out a way to do it in a way more conducive to maintenance.

**Review by nikitawootten-nist — APPROVED** [review]

**Comment by Compton-US** [comment]
I'm going to click approve, because I think we should move forward and we can always adjust in the future.  

I wanted to record a thought though:
The main page could provide initial direction for locations of the model documentation, tutorials, github projects, etc.  I'm thinking of formats similar to [pandas](https://pandas.pydata.org/), [scipy](https://docs.scipy.org/doc/scipy/) and [numpy](https://numpy.org/doc/stable/).  For example, numpy provides direction to "Getting Started", "User Guide", "API Reference" and "Contributors Guide" as the content of the landing page.  We might benefit from a similar format.

**Review by Compton-US — APPROVED** [review]

**Review by iMichaela — APPROVED** [review]
For now, what we have on the page is very good - much better than the 'nothing' we had before. I like Chris' suggestions but for next round of improvements.

**Pull request merged**